src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star:last-child {
+  margin-bottom: 0;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
It can be simplified
`background: url("images/star.svg") no-repeat center;`

Reply:
Oh, thanks!